src/common/types/store-data/idev-tool-state.d.ts

```diff
@@ -4,4 +4,5 @@ export interface DevToolState {
     isOpen: boolean;
     inspectElement?: string[];
     frameUrl?: string;
+    count?: number;
```

Comment:
Consider not making `count` optional in the interface, since it's already always there in non-test code (requires including it in stub in frame-url-search-initiator.test.ts)

Reply:
updated!